src/text_file_functions.cpp

```diff
@@ -1260,12 +1260,12 @@ bool AOApplication::get_player_count_optout()
 
 bool AOApplication::get_sfx_on_idle()
 {
-  return configini->value("sfx_on_idle", "false").value<QString>()
+  return configini->value("sfx_on_idle", "true").value<QString>()
```

Comment:
No.

Reply:
This will break almost *all* pre-existing content and cause immense confusion among the existing player base, most of whom will go and turn it off anyway. I don't know why you're fixated on this, but you can just distribute a default settings file with your own server content instead of doing this. Please do not do this.